app/scripts/components/table/list-view.js

```diff
@@ -231,11 +233,16 @@ class List extends React.Component {
           order={order}
           changeSortProps={this.queryNewSort}
           onSelect={this.updateSelection}
-          canSelect={!!hasActions}
+          canSelect={hasActions}
```

Comment:
I assume that the double bang was to cast this as a boolean? Is it already a boolean? I guess it must be or else the props validation on SortableTable would fail when the prop is passed

Reply:
See my reply above about the addition of `> 0`. Regarding validation, using `!!` is required to coerce `hasActions` to a boolean value (specifically to `true` when `hasActions` is _truthy_; `false` otherwise) because without the addition of `> 0` above, it is an integer value (specifically the length of the `bulkActions` array).